docs/conf.py

```diff
@@ -25,7 +25,13 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.viewcode']
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.intersphinx',
+    'repoze.sphinx.autointerface',
```

Comment:
Maybe these things should be in the `docs` extra that already includes Sphinx? And then tox could install `.[docs]`.